scipy/spatial/tests/test_qhull.py

```diff
@@ -701,6 +701,43 @@ def test_volume_area(self):
         assert_allclose(tri.volume, 1., rtol=1e-14)
         assert_allclose(tri.area, 6., rtol=1e-14)
 
+    def test_good2d(self):
+        # Make sure the QGn option gives the correct value of "good".
+        points = np.array([[0.2, 0.2],
+                           [0.2, 0.4],
+                           [0.4, 0.4],
+                           [0.4, 0.2],
+                           [0.3, 0.6]])
+        hull = ConvexHull(points=points, qhull_options='QG4')
```

Comment:
if you check the CI results and test examples above, you'll see that `qhull.ConvexHull` is needed here and in the other cases you've added

Reply:
I have no idea how to read that.